mllib/src/test/scala/org/apache/spark/ml/PipelineSuite.scala

```diff
@@ -92,6 +93,7 @@ class PipelineSuite extends SparkFunSuite {
     val copied = model.copy(ParamMap(hashingTF.numFeatures -> 10))
     require(copied.stages(0).asInstanceOf[HashingTF].getNumFeatures === 10,
       "copy should handle extra stage params")
+    MLTestingUtils.checkCopy(model)
```

Comment:
This should be in a different test.  The model should be produced via fit(), not constructed from scratch, so that the model has a parent.